src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -792,6 +795,38 @@
             };
         }
 
+        protected virtual void CheckIncompatibleRules(IEnumerable<Skimmer<TContext>> skimmers, TContext context, ISet<string> disabledSkimmers)
```

Comment:
>protected [](http://example.com/codeflow?start=8&length=9)

Can you shut down accessibility of this method for now? i.e., mark it private

This class is pretty carefully constructed to follow a sequence of specific overridable methods to function.

I understand why you made it protected virtual, because imposing exist is pretty serious. :) But let's leave it until we can do a more thorough review of the way this API is evolving (i.e., we should be cautious about introducing new exposed methods).

Also, can you tell me, does your new functionality work for the multithreaded command as well?

Reply:
Understand, changed to internal so it can be accessed by the unit tests.

The same code and tests is added for `MultithreadedAnalyzeCommandBase`, could not find a common place for it, thinking about extract the common code of these 2 classes into a common base class.